cypress/integration/chi-js/chi-drawer.spec.js

```diff
@@ -0,0 +1,127 @@
+describe('chi-drawer', function() {
+  before(() => {
+    cy.visit('/tests/js/chi-drawer.html');
+  });
+
+  describe('Test open & close behavior of Left positioned Drawer', () => {
+    it('Click on Drawer trigger opens the Drawer', () => {
+      cy.get('#drawer-trigger-1').click();
+      cy.get('#drawer-1')
+        .should('have.class', '-active')
+        .should('have.class', '-left')
```

Comment:
```suggestion
```
This is checking if the class is present in HTML markup.
The proper test would be to check if `left: 0;` CSS is present.
Please use
`.should('have.css' . . .);`